src/plot_api/plot_api.js

```diff
@@ -2124,6 +2126,7 @@ Plotly.update = function update(gd, traceUpdate, layoutUpdate, traces) {
             data: restyleSpecs.eventData,
             layout: relayoutSpecs.eventData
         });
+        gd.emit('plotly_plotmodified');
```

Comment:
> Should afterplot be emitted between frames?

Hmm. Very good question. I'd say _no_. `plotly_afterplot` should be the fired just before the promise is returned. But, I can be convinced otherwise.